### PR TITLE
Update static html files

### PR DESCRIPTION
**Pull request by Robinlovelace**

### CODE REVIEW

File: static/about.Rmd [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+In summary the PCT is a planning support system to improve cycling provision at many levels from regions to specific points on the road network. For further work on the approach, please see the [academic paper](http://arxiv.org/abs/1509.04425): The Propensity to Cycle Tool: An open source online system for sustainable transport planning [@lovelace_propensity_2015].
+
+We would like to thank Brook Lyndhurst for facilitating Phase 1 of the DfT contract and Atkins for facilitating Phase 2.
+
+We would also like to thank the ESRC and EPSRC for additional funding through Impact Acceleration Awards.
+
+The PCT uses transparent and tested methods on the best available available data. However, we cannot accept liability for, included but not limited to, loss or damages.
+
+For more information or questions, please contact us at: [npct@npct.org.uk](mailto:npct@npct.org.uk).
```
Comment (AnnaGoodman1) [inline static/about.Rmd]: change email to pct@pct.bike

File: static/about.Rmd [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Central both to strategic and smaller-scale use is the question of where to prioritise high quality cycling infrastructure of sufficient capacity for a planned growth in cycling.
+
+In summary the PCT is a planning support system to improve cycling provision at many levels from regions to specific points on the road network. For further work on the approach, please see the [academic paper](http://arxiv.org/abs/1509.04425): The Propensity to Cycle Tool: An open source online system for sustainable transport planning [@lovelace_propensity_2015].
+
+We would like to thank Brook Lyndhurst for facilitating Phase 1 of the DfT contract and Atkins for facilitating Phase 2.
+
+We would also like to thank the ESRC and EPSRC for additional funding through Impact Acceleration Awards.
+
+The PCT uses transparent and tested methods on the best available available data. However, we cannot accept liability for, included but not limited to, loss or damages.
```
Comment (AnnaGoodman1) [inline static/about.Rmd]: remove double word 'available'

File: static/about.Rmd [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+- [Alvaro Ullrich](http://www.cedar.iph.cam.ac.uk/people/support/alvaro-ullrich/)
+- [Ali Abbas](http://www.cedar.iph.cam.ac.uk/people/cdfs/ali-abbas/)
+- [Nikolai Berkoff](https://github.com/nikolai-b)
+
+The Propensity to Cycle Tool (PCT) was designed to assist transport planners and policy makers to prioritise investments and interventions to promote cycling. The PCT answers the question: "where is cycling currently common and where does cycling have the greatest potential to grow?"
+
+The PCT can be used at different scales.
+
+First, the PCT is a strategic planning tool. Different visons of the future are represented through the various scenarios of change, including the government's target to double cycling in a decade to the more ambitious 'Go Dutch' scenario whereby Dutch propensities to cycle (allowing for English hilliness and trip distances) are reached in England. By showing what the rate of cycling could feasibly look like in different parts of cities and regions, and illustrating the associated increase in cycle use on the road network, the PCT should inform policies that seek a wider shift towards sustainable transport more generally.
```
Comment (AnnaGoodman1) [inline static/about.Rmd]: change 'to the more ambitious 'Go Dutch'' to 'and the more ambitious 'Go Dutch''

File: static/about.Rmd (hunk 2 above)
Comment (RachelAldred) [inline static/about.Rmd]: included should be 'including'

File: static/about.Rmd (hunk 1 above)
Comment (Robinlovelace) [inline static/about.Rmd]: Thanks @AnnaGoodman1 - updated now

File: static/about.Rmd (hunk 3 above)
Comment (Robinlovelace) [inline static/about.Rmd]: Job done :-)

File: static/about.Rmd (hunk 2 above)
Comment (Robinlovelace) [inline static/about.Rmd]: Thanks for the comment @RachelAldred - I think I've fixed it now - pushed to the test site.

**Pull request merged**